integrations/oauth/configs.go

```diff
@@ -671,16 +671,20 @@ func setupAuth0(vs sdktypes.Vars, cfg *oauthConfig) {
 
 // TODO(INT-349): Support GHES in private OAuth mode too.
 func privatizeGitHub(vs sdktypes.Vars, cfg *oauthConfig) {
-	defaultAppName := os.Getenv("GITHUB_APP_NAME")
-	if defaultAppName == "" {
-		defaultAppName = defaultGitHubAppName
+	enterpriseURL := vs.GetValueByString("enterprise_url")
+	if enterpriseURL == "" {
+		enterpriseURL = defaultGitHubBaseURL
```

Comment:
Not enough, you should also support the env var `GITHUB_ENTERPRISE_URL` in case `vs.GetValueByString("enterprise_url")` is empty, and use `defaultGitHubBaseURL` only if both of them are empty.